src/resources/packages/classy/types/store.ts

```diff
@@ -0,0 +1,25 @@
+export interface TicketState {
```

Comment:
Again about conventions: I've gone with one-type-one-file convention so far in _most_ of the cases, and stuck with `CamelCase.d.ts` file names.
I'm neutral toward file naming and grouping multiple types in one type definition file; let's agree on a convention and stick with it in all stores and in all Classy packages.

Reply:
I like the current convention:

> I've gone with one-type-one-file convention so far in most of the cases, and stuck with `CamelCase.d.ts` file names.

I think this is easy to understand where to find types, and the camel case helps to distinguish types when using them in the code. So I think we should keep using this same convention.

(This file will change as part of cleaning up the store)